test/unit/lib/plugins/aws/package/compile/events/apiGateway/lib/usagePlan.test.js

```diff
@@ -363,6 +363,28 @@ describe('UsagePlan', () => {
     });
   });
 
+  it('Should have values for throttle and not quota', () => {
+    serverlessConfigurationExtension.provider.apiGateway = {
+      usagePlan: { throttle, quota: { limit: null, offset: null, period: null } },
+    };
+    return runServerless({
```

Comment:
Let's use `async/await` here as we're trying to migrate from promise-centered syntax across the codebase

Reply:
Hi @pgrzesik, thank you for the feedback :pray:, how's this for `async/await`:

```
it('Should have values for throttle and not quota', async () => {
    serverlessConfigurationExtension.provider.apiGateway = {
      usagePlan: { throttle, quota: { limit: null, offset: null, period: null } },
    };
    const cfResources = await runServerless({
      fixture: 'apiGateway',
      configExt: serverlessConfigurationExtension,
      cliArgs: ['package'],
    }).then(({cfTemplate}) => cfTemplate.Resources)

    expect(cfResources.ApiGatewayUsagePlan.Properties.Throttle.BurstLimit).to.be.equal(
      burstLimit
...
```

If you're happy with that method, then I'm happy to update the rest of the tests too.

Reply:
It should look a little bit different - more like:

```
const { cfTemplate } = await runServerless(...);
expect(cfTemplate.Resources).to......
```

Reply:
Thanks - that's better for sure! I've made the changes 👍